index.html

```diff
@@ -45,7 +45,7 @@
     </style>
 
 <script>
-    const minTime = 10000; // So guests can enjoy your welcome page before being redirected.
+    const minTime = 3000; // So guests can enjoy your welcome page before being redirected.
```

Comment:
I did some quick research and I think I may be able to make this a container variable. I must have just messed it up before: see https://stackoverflow.com/questions/4870328/read-environment-variables-in-node-js.

Reply:
Yeah, I think it would be something along the lines of this: https://stackoverflow.com/questions/51044328/pass-docker-variable-to-react-index-html-file

Reply:
Yeah, that's kinda what I was thinking.

Reply:
I'll mark this as resolved now, and follow up with that later.